serverless/serverless
# Added ability to add S3 deployment resource tags

## What did you implement:
Ability to add tags to the AWS S3 deployment resources created by serverless
for https://github.com/serverless/serverless/issues/3817



## How did you implement it:
Added function to uploadArtifacts.js to check for existence of `tags` key on `deploymentBucket` object, and if so, loop through all the tags and create a url encoded string to add to S3 upload object that will be added to each deployed resource item



## How can we verify it:
Add the `tags` key to `deploymentBucket` as shown in the updated docs, then run the deploy command, and check if deployed resources have the specified tags in S3



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
